src/main/java/io/omnition/loadgenerator/App.java

```diff
@@ -29,6 +29,9 @@
     @Parameter(names = "--paramsFile", description = "Name of the file containing the topology params", required = true)
     private String topologyFile;
 
+    @Parameter(names = "--logLevel", description = "Level of log verbosity (1..2). If omitted defaults to 2, most verbose output.", required = false)
+    private Integer logLevel = 2;
```

Comment:
I kept default level at 2 to make the default behaviour the same as current behaviour. However, if you think it is not critical to keep backwards compatibility I would make the default level 1.